onedal/common/backend/utils.h

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <numpy/arrayobject.h>
+
+#define ONEDAL_2021_3_VERSION (2021 * 10000 + 3 * 100)
+
+#ifdef _WIN32
+    #define NOMINMAX
+    #define ONEDAL_BACKEND_EXPORT __declspec(dllexport)

```

Comment:
We have `ONEDAL_EXPORT` in oneDAL. Why we cannot use it here?

Reply:
In this case, symbols from daal's headers will try to be exported in python dll. from this problem an additional define was made `__ONEDAL_ENABLE_DLL_EXPORT__`  which only works when compiling the oneDAL library